src/assets/locales/fr.json

```diff
@@ -531,8 +593,19 @@
   "tips.mobile.locomotion": "Bravo ! Pour vous déplacer, pincez avec deux doigts.",
   "tips.mobile.look": "Bienvenue ! 👋 Cliquez et faites glisser pour regarder autour de vous.",
   "tips.streaming": "Diffusion en cours vers le hall d'attente. Pour quitter le mode diffusion, utilisez le menu Plus quand vous aurez terminé.",
+  "token-type.account-token": "Jeton Compte",
+  "token-type.app-token": "Jeton App",
```

Comment:
```suggestion
  "token-type.app-token": "Jeton d'app",
```

Reply:
Ok, I did the change in a separate commit.